docs/wifi-menu.8.txt

```diff
@@ -0,0 +1,89 @@
+WIFI-MENU(8)
+============
+
+NAME
+----
+wifi-menu - Interactively connect to a wireless network
+
+
+SYNOPSIS
+--------
+*wifi-menu* [-o | --obscure] [+INTERFACE+]
+
+*wifi-menu* [-h | --help]
+
+
+DESCRIPTION
+-----------
+*wifi-menu* allows a user to interactively connect to a wireless network
+over INTERFACE using an existing netctl profile, or an automatically
+generated profile for basic configurations. If only one wireless
```

Comment:
The fact that only basic configurations are supported for generating profiles is already stressed in the next paragraph. No need to state it twice.
```suggestion
on INTERFACE using a netctl profile. In case no netctl profile is available, one can be generated.
If only one wireless
```
(pending reflow to the proper number of columns)